tools/minify.cpp

```diff
@@ -5,74 +5,50 @@
 #include <unistd.h>
 
 #include "simdjson.h"
+#include "cxxopts.hpp"
 
-// Stash the exe_name in main() for functions to use
-char* exe_name;
+cxxopts::Options options("minify", "Runs the parser against the given json files in a loop, measuring speed and other statistics.");
 
-void print_usage(std::ostream& out) {
-  out << "Usage: " << exe_name << "  [-a ARCH] <jsonfile>" << std::endl;
-  out << std::endl;
-  out << "Runs the parser against the given json files in a loop, measuring speed and other statistics." << std::endl;
-  out << std::endl;
-  out << "Options:" << std::endl;
-  out << std::endl;
-  out << "-a IMPL      - Use the given parser implementation. By default, detects the most advanced" << std::endl;
-  out << "               implementation supported on the host machine." << std::endl;
+void usage(std::string message) {
+  std::cerr << message << std::endl;
+  std::cerr << options.help() << std::endl;
+  std::cerr << "Available parser implementations:" << std::endl;
   for (auto impl : simdjson::available_implementations) {
-    out << "-a " << std::left << std::setw(9) << impl->name() << " - Use the " << impl->description() << " parser implementation." << std::endl;
+    std::cerr << "-a " << std::left << std::setw(9) << impl->name() << " - Use the " << impl->description() << " parser implementation." << std::endl;
```

Comment:
Is there a way to make this part of the help for the `-a` option?

Reply:
Unfortunately, it looks like there is a formatting problem when the description of an option is too complex (with line breaks), which removes much of the text..

Reply:
> there is a formatting problem

Can you elaborate? In a PR to your own PR, it appears that I was able to do as @jkeiser  suggests....

https://github.com/yoannlr/simdjson_cxxopts_tools/pull/1